core/templates/dev/head/domain/exploration/StatesObjectFactorySpec.ts

```diff
@@ -19,13 +19,18 @@
 describe('States object factory', function() {
   beforeEach(angular.mock.module('oppia'));
 
+  var oldValueForNewStateTemplate = null;
+
   describe('StatesObjectFactory', function() {
     var scope, sof, ssof, statesDict, statesWithAudioDict, atof;
-    beforeEach(angular.mock.inject(function($injector) {
+
+    beforeEach(inject(function($injector) {
```

Comment:
Hmm so I'm confused, test files are exceptional and can use plain 'ol `$inject` instead of `angular.mock.inject`?

Reply:
Hi @brianrodri, yes actually typescript compiler only produces error when `module` is instead of `angular.mock.module` since angular-mock typings does not have definition for `module` but for inject it works correctly. It is better to follow the same behavior for inject though and I think we will need to add a test for the same.

Reply:
Created an issue: https://github.com/oppia/oppia/issues/6678

Reply:
Sorry, not sure how that got missed. Made the updates!